sci-libs/factory/factory-3.1.1-r2.ebuild

```diff
@@ -39,4 +37,12 @@ src_configure() {
 	autotools-utils_src_configure
 }
 
-# TODO: get rid of factories static libs ?
+src_install() {
+	autotools-utils_src_install
+
+	# Passing --disable-static to configure won't disable the build of static libs,
+	# as libtool isn't used. Therefore the static libs are deleted after installed to ${D}.
+	if ! use static-libs; then
+		find "${D}" -type f -name "*.a" -delete || die
```

Comment:
I wasn't aware of ${ED}, as it is not documented in [variables section](http://devmanual.gentoo.org/ebuild-writing/variables/). I also think that using ${ED}=${D%/}${EPREFIX}/ or using ${D} directly doesn't matter in this case. Anyway, I adjusted the ebuild. Thanks for the advice.